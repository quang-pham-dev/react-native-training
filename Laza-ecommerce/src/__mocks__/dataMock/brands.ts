/* A data array. */

export const brand = {
  id: '0',
  name: 'Nike',
  logoUrl:
    'https://toppng.com/uploads/preview/ike-logo-png-hd-nike-logo-png-image-free-download-nike-logo-sv-11562865034fkhqkcoas6.png'
};

export const brands = [
  {
    id: '0',
    name: 'Nike',
    logoUrl:
      'https://toppng.com/uploads/preview/ike-logo-png-hd-nike-logo-png-image-free-download-nike-logo-sv-11562865034fkhqkcoas6.png'
  },
  {
    id: '1',
    name: 'Adidas',
    logoUrl:
      'https://www.pikpng.com/pngl/m/42-429866_adidas-logo-png-images-free-download-rh-pngimg.png'
  },
  {
    id: '2',
    name: 'Fila',
    logoUrl:
      'https://toppng.com/uploads/preview/fila-logo-vector-download-free-11574216707jcmvfu58fy.png'
  },
  {
    id: '3',
    name: 'Puma',
    logoUrl:
      'https://banner2.cleanpng.com/20180613/gsj/kisspng-tracksuit-puma-logo-brand-clothing-puma-logo-5b20dc6b232426.487874081528880235144.jpg'
  },
  {
    id: '4',
    name: 'Gucci',
    logoUrl: 'https://esquilo.io/png/thumb/XBwczcMncYeff65-Vector-Gucci-Logo-PNG-Free-Download.png'
  }
];
