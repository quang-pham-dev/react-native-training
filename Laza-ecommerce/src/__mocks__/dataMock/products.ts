import { ProductProps } from 'types/Products';

export const product: ProductProps = {
  id: '0',
  brandId: '1',
  name: 'Nike',
  price: 100,
  type: 'shoes',
  like: true,
  title: 'Nike Sportswear Club Fleece',
  source:
    'https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80',
  description:
    "Nike is a multinaelichick and retired quarterback Jack Nicklaus. Nike's products include apparel, shoes, equipment, and accessories. Nike also owns and operates the majority of the world's sporting goods brands, including Adidas, Puma, and Fila.",
  reviewer: {
    id: '2',
    name: 'John Doe',
    date: '13, Sep, 2020',
    image:
      'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
  },
  imageReview: [
    {
      id: '0',
      image:
        'https://images.unsplash.com/photo-1499852848443-3004d6dc4cfc?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1374&q=80',
    },
    {
      id: '1',
      image:
        'https://images.unsplash.com/photo-1618898909019-010e4e234c55?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80',
    },
    {
      id: '2',
      image:
        'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
    },
    {
      id: '3',
      image:
        'https://images.unsplash.com/photo-1518656306295-aa28b28b2504?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
    },
    {
      id: '4',
      image:
        'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
    },
    {
      id: '5',
      image:
        'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
    },
  ],
  rating: '4.5',
  comment:
    'lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,adipisicing elit. Quisquam, quia.',
};

export const products: ProductProps[] = [
  {
    id: '0',
    brandId: '1',
    name: 'Nike',
    price: 100,
    type: 'shoes',
    like: true,
    title: 'Nike Sportswear Club Fleece',
    source:
      'https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80',
    description:
      "Nike is a multinaelichick and retired quarterback Jack Nicklaus. Nike's products include apparel, shoes, equipment, and accessories. Nike also owns and operates the majority of the world's sporting goods brands, including Adidas, Puma, and Fila.",
    reviewer: {
      id: '2',
      name: 'John Doe',
      date: '13, Sep, 2020',
      image:
        'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
    },
    imageReview: [
      {
        id: '0',
        image:
          'https://images.unsplash.com/photo-1499852848443-3004d6dc4cfc?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1374&q=80',
      },
      {
        id: '1',
        image:
          'https://images.unsplash.com/photo-1618898909019-010e4e234c55?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80',
      },
      {
        id: '2',
        image:
          'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
      {
        id: '3',
        image:
          'https://images.unsplash.com/photo-1518656306295-aa28b28b2504?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
      {
        id: '4',
        image:
          'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
      },
      {
        id: '5',
        image:
          'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
    ],
    rating: '4.5',
    comment:
      'lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,adipisicing elit. Quisquam, quia.',
  },
  {
    id: '0',
    brandId: '1',
    name: 'Nike',
    price: 100,
    type: 'shoes',
    like: true,
    title: 'Nike Sportswear Club Fleece',
    source:
      'https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80',
    description:
      "Nike is a multinaelichick and retired quarterback Jack Nicklaus. Nike's products include apparel, shoes, equipment, and accessories. Nike also owns and operates the majority of the world's sporting goods brands, including Adidas, Puma, and Fila.",
    reviewer: {
      id: '2',
      name: 'John Doe',
      date: '13, Sep, 2020',
      image:
        'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
    },
    imageReview: [
      {
        id: '0',
        image:
          'https://images.unsplash.com/photo-1499852848443-3004d6dc4cfc?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1374&q=80',
      },
      {
        id: '1',
        image:
          'https://images.unsplash.com/photo-1618898909019-010e4e234c55?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80',
      },
      {
        id: '2',
        image:
          'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
      {
        id: '3',
        image:
          'https://images.unsplash.com/photo-1518656306295-aa28b28b2504?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
      {
        id: '4',
        image:
          'https://images.unsplash.com/photo-1609505848912-b7c3b8b4beda?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=765&q=80',
      },
      {
        id: '5',
        image:
          'https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470',
      },
    ],
    rating: '4.5',
    comment:
      'lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,adipisicing elit. Quisquam, quia.',
  },
];
