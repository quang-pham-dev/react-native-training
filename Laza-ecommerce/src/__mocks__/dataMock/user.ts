export const user = {
  id: '621c4188dece66a40dad2951',
  avatar: 'https://picsum.photos/200',
  username: 'Quangpham',
  gender: 'male',
  email: 'admin@admin.com',
  phone: '+84 (919) 507231',
  address: '199 Bergen Street, Bowie, Oregon, 7933',
  role: 'admin',
  password: '123456'
};
