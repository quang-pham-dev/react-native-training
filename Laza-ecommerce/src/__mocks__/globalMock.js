import mockAsyncStorage from '@react-native-async-storage/async-storage/jest/async-storage-mock';

jest.mock('@react-native-community/async-storage', () => mockAsyncStorage);
