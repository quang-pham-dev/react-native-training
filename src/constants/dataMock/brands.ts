export const brand = {
  id: '0',
  name: 'Nike',
  url: 'https://1000logos.net/wp-content/uploads/2021/11/Nike-Logo.png',
}

export const brands = [
  {
    id: '0',
    name: 'Nike',
    url: 'https://1000logos.net/wp-content/uploads/2021/11/Nike-Logo.png',
  },
  {
    id: '1',
    name: 'Adidas',
    url: 'https://www.pikpng.com/pngl/m/42-429866_adidas-logo-png-images-free-download-rh-pngimg.png',
  },
  {
    id: '2',
    name: 'Fila',
    url: 'https://toppng.com/uploads/preview/fila-logo-vector-download-free-11574216707jcmvfu58fy.png',
  },
  {
    id: '3',
    name: 'Puma',
    url: 'https://banner2.cleanpng.com/20180613/gsj/kisspng-tracksuit-puma-logo-brand-clothing-puma-logo-5b20dc6b232426.487874081528880235144.jpg',
  },
  {
    id: '4',
    name: 'Gucci',
    url: 'https://esquilo.io/png/thumb/XBwczcMncYeff65-Vector-Gucci-Logo-PNG-Free-Download.png',
  },
]
