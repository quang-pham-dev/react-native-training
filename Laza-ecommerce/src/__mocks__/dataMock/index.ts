import { user } from './user';
import { brand, brands } from './brands';
import { product, products } from './products';

export { brand, brands, product, products, user };
