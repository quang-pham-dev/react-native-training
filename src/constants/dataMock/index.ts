import {user} from './user'
import {brand, brands} from './brands'

export {brand, brands, user}
