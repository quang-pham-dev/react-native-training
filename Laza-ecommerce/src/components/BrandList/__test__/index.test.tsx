import React from 'react';
import BrandsCardList from 'components/BrandList';
import { brandsData } from '__mocks__/dataMock';
import renderer from 'react-test-renderer';
import { TouchableOpacity } from 'react-native';

describe('Brand Card List Component', () => {
  const navigate = jest.fn();

  const component = renderer.create(
    <BrandsCardList handleNavigationBrandDetailScreen={navigate} brandsData={brandsData} />,
  );
  test('Should render correctly', () => {
    const result = component.toJSON();
    expect(result).toMatchSnapshot();
  });

  test('should call function handleNavigationBrandDetailScreen', () => {
    const press = component.root.findAllByType(TouchableOpacity)[0];
    press.props.onPress();
    expect(navigate).toHaveBeenCalled();
  });
});
