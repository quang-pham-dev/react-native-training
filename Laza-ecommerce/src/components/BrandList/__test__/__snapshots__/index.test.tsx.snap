// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand Card List Component Should render correctly 1`] = `
<View
  style={Object {}}
>
  <RCTScrollView
    ListEmptyComponent={[Function]}
    data={
      Array [
        Object {
          "id": "0",
          "logoUrl": "https://www.adidas.com/on/demandware.static/-/Sites-adidas-US-Library/default/dwf9f9f9f/images/logo.png",
          "name": "Adidas",
        },
        Object {
          "id": "1",
          "logoUrl": "https://www.nike.com/images/t_PDP_1280_v1/b_white/f_auto/j_pdp_white/k_P/w_80/0f0f0f/0f0f0f_nike_logo_white.png",
          "name": "Nike",
        },
        Object {
          "id": "2",
          "logoUrl": "https://www.fila.com/on/demandware.static/-/Sites-fila-US-Library/default/dwf9f9f9f/images/logo.png",
          "name": "Fila",
        },
        Object {
          "id": "3",
          "logoUrl": "https://www.puma.com/on/demandware.static/-/Sites-puma-US-Library/default/dwf9f9f9f/images/logo.png",
          "name": "Puma",
        },
        Object {
          "id": "4",
          "logoUrl": "https://www.gucci.com/on/demandware.static/-/Sites-gucci-US-Library/default/dwf9f9f9f/images/logo.png",
          "name": "Guci",
        },
      ]
    }
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={true}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    showsHorizontalScrollIndicator={false}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View>
      <View
        onLayout={[Function]}
        style={
          Array [
            Object {
              "flexDirection": "row",
            },
            null,
          ]
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "justifyContent": "center",
              "opacity": 1,
            }
          }
          testID="brandCard"
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#F5F6FA",
                  "borderRadius": 10,
                  "flexDirection": "row",
                  "height": 50,
                  "marginRight": 10,
                  "paddingRight": 10,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "backgroundColor": "#fff",
                  "borderRadius": 10,
                  "height": 40,
                  "marginLeft": 5,
                  "marginRight": 10,
                  "marginVertical": 5,
                  "padding": 8,
                  "width": 40,
                }
              }
            >
              <Image
                source={
                  Object {
                    "uri": "https://www.adidas.com/on/demandware.static/-/Sites-adidas-US-Library/default/dwf9f9f9f/images/logo.png",
                  }
                }
                style={
                  Object {
                    "height": "100%",
                    "resizeMode": "contain",
                    "width": "100%",
                  }
                }
              />
            </View>
            <View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#1D1E20",
                      "fontSize": 17,
                    },
                    Object {
                      "color": "#1D1E20",
                      "fontFamily": "Inter_500Medium",
                      "fontSize": 15,
                      "lineHeight": 16.5,
                    },
                  ]
                }
              >
                Adidas
              </Text>
            </View>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={
          Array [
            Object {
              "flexDirection": "row",
            },
            null,
          ]
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "justifyContent": "center",
              "opacity": 1,
            }
          }
          testID="brandCard"
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#F5F6FA",
                  "borderRadius": 10,
                  "flexDirection": "row",
                  "height": 50,
                  "marginRight": 10,
                  "paddingRight": 10,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "backgroundColor": "#fff",
                  "borderRadius": 10,
                  "height": 40,
                  "marginLeft": 5,
                  "marginRight": 10,
                  "marginVertical": 5,
                  "padding": 8,
                  "width": 40,
                }
              }
            >
              <Image
                source={
                  Object {
                    "uri": "https://www.nike.com/images/t_PDP_1280_v1/b_white/f_auto/j_pdp_white/k_P/w_80/0f0f0f/0f0f0f_nike_logo_white.png",
                  }
                }
                style={
                  Object {
                    "height": "100%",
                    "resizeMode": "contain",
                    "width": "100%",
                  }
                }
              />
            </View>
            <View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#1D1E20",
                      "fontSize": 17,
                    },
                    Object {
                      "color": "#1D1E20",
                      "fontFamily": "Inter_500Medium",
                      "fontSize": 15,
                      "lineHeight": 16.5,
                    },
                  ]
                }
              >
                Nike
              </Text>
            </View>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={
          Array [
            Object {
              "flexDirection": "row",
            },
            null,
          ]
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "justifyContent": "center",
              "opacity": 1,
            }
          }
          testID="brandCard"
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#F5F6FA",
                  "borderRadius": 10,
                  "flexDirection": "row",
                  "height": 50,
                  "marginRight": 10,
                  "paddingRight": 10,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "backgroundColor": "#fff",
                  "borderRadius": 10,
                  "height": 40,
                  "marginLeft": 5,
                  "marginRight": 10,
                  "marginVertical": 5,
                  "padding": 8,
                  "width": 40,
                }
              }
            >
              <Image
                source={
                  Object {
                    "uri": "https://www.fila.com/on/demandware.static/-/Sites-fila-US-Library/default/dwf9f9f9f/images/logo.png",
                  }
                }
                style={
                  Object {
                    "height": "100%",
                    "resizeMode": "contain",
                    "width": "100%",
                  }
                }
              />
            </View>
            <View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#1D1E20",
                      "fontSize": 17,
                    },
                    Object {
                      "color": "#1D1E20",
                      "fontFamily": "Inter_500Medium",
                      "fontSize": 15,
                      "lineHeight": 16.5,
                    },
                  ]
                }
              >
                Fila
              </Text>
            </View>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={
          Array [
            Object {
              "flexDirection": "row",
            },
            null,
          ]
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "justifyContent": "center",
              "opacity": 1,
            }
          }
          testID="brandCard"
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#F5F6FA",
                  "borderRadius": 10,
                  "flexDirection": "row",
                  "height": 50,
                  "marginRight": 10,
                  "paddingRight": 10,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "backgroundColor": "#fff",
                  "borderRadius": 10,
                  "height": 40,
                  "marginLeft": 5,
                  "marginRight": 10,
                  "marginVertical": 5,
                  "padding": 8,
                  "width": 40,
                }
              }
            >
              <Image
                source={
                  Object {
                    "uri": "https://www.puma.com/on/demandware.static/-/Sites-puma-US-Library/default/dwf9f9f9f/images/logo.png",
                  }
                }
                style={
                  Object {
                    "height": "100%",
                    "resizeMode": "contain",
                    "width": "100%",
                  }
                }
              />
            </View>
            <View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#1D1E20",
                      "fontSize": 17,
                    },
                    Object {
                      "color": "#1D1E20",
                      "fontFamily": "Inter_500Medium",
                      "fontSize": 15,
                      "lineHeight": 16.5,
                    },
                  ]
                }
              >
                Puma
              </Text>
            </View>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={
          Array [
            Object {
              "flexDirection": "row",
            },
            null,
          ]
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "justifyContent": "center",
              "opacity": 1,
            }
          }
          testID="brandCard"
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#F5F6FA",
                  "borderRadius": 10,
                  "flexDirection": "row",
                  "height": 50,
                  "marginRight": 10,
                  "paddingRight": 10,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "backgroundColor": "#fff",
                  "borderRadius": 10,
                  "height": 40,
                  "marginLeft": 5,
                  "marginRight": 10,
                  "marginVertical": 5,
                  "padding": 8,
                  "width": 40,
                }
              }
            >
              <Image
                source={
                  Object {
                    "uri": "https://www.gucci.com/on/demandware.static/-/Sites-gucci-US-Library/default/dwf9f9f9f/images/logo.png",
                  }
                }
                style={
                  Object {
                    "height": "100%",
                    "resizeMode": "contain",
                    "width": "100%",
                  }
                }
              />
            </View>
            <View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#1D1E20",
                      "fontSize": 17,
                    },
                    Object {
                      "color": "#1D1E20",
                      "fontFamily": "Inter_500Medium",
                      "fontSize": 15,
                      "lineHeight": 16.5,
                    },
                  ]
                }
              >
                Guci
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
